app/views/layouts/application.html.erb

```diff
@@ -28,12 +28,12 @@
     <li class="nav-li"><%= link_to "ABOUT", about_path %></li>
     <li class="dropdown-parent nav-li"><%= link_to "ARTISTS", categories_path %>
       <div class="dropdown-body">
-        <%= link_to "ORIGINAL ART" %>
-        <%= link_to "PRINTS" %>
-        <%= link_to "STATIONERY" %>
-        <%= link_to "JEWELRY" %>
-        <%= link_to "WEARABLES" %>
-        <%= link_to "HOME & SELF CARE" %>
+        <%= link_to "ORIGINAL ART", '/categories/1' %>
```

Comment:
why are they numbered? does it make more sense for the routes to be human-parsable?